package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "fh-mbaas-api",
-  "version": "5.13.1",
+  "version": "5.13.2",
```

Comment:
I think we should establish a more reasonable way to release modules, and to version bumps at release time /cc @grdryn